configure.ac

```diff
@@ -29,19 +32,45 @@ AC_ARG_ENABLE(
   [signature-verification],
   [AS_HELP_STRING([--enable-signature-verification], [Enable signature check (disabled by default)])],
   [AC_DEFINE([SIGNATURES], 1, [Enable signature check as default])]
-
+    SIGVERIFICATION=${enableval}
```

Comment:
I would do the same thing here, just read the "enable_signature_verification" variable to decide what to do later in the script.